testsuite/features/init_clients/srv_create_activationkey.feature

```diff
@@ -9,13 +9,28 @@ Feature: Create activation keys
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
+  Scenario: Clone the child custom channel including test repositories
```

Comment:
We need Cucumber tags in SLE scenarios, as we do for rhlike_minion scenarios.

Reply:
I'll have to check if we have corresponding tags everywhere it is referenced
It is also used for ssh_minion